src/components/Main.js

```diff
@@ -87,17 +83,13 @@ export default class Main extends React.Component {
   }
 }
 
-function injectCSS (url) {
-  var link = document.createElement('link');
-  link.href = url;
-  link.type = 'text/css';
-  link.rel = 'stylesheet';
-  link.media = 'screen,print';
-  link.setAttribute('data-aframe-editor', 'style');
```

Comment:
I've never used webfont, but maybe you can inject somehow the attributes on the code that it will generate?